zuul.d/jobs.yaml

```diff
@@ -41,3 +41,15 @@
     description: Install and run pre-commit
     parent: base
     run: playbooks/pre-commit.yaml
+
+- job:
+    name: oc-cluster-up
+    parent: base
+    attempts: 1
+    description: Deploy Openshift v3 cluster
+    pre-run:
+      - playbooks/oc-cluster-up.yaml
+    nodeset:
+      nodes:
+        - name: test-node
+          label: cloud-fedora-30
```

Comment:
did 31 already have the cgroups v2 change?

Reply:
Yes, (https://github.com/packit/packit-service/pull/381  & https://github.com/packit/packit-service/pull/749#issuecomment-662971892)